archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -0,0 +1,404 @@
+     optdepends=('llvm-git: for the man file')
+     provides=("lldb=$pkgver")
+     conflicts=('lldb')
+     _fakeinstall_64 fakeinstall_64/usr/bin/lldb*
+     _fakeinstall_64 fakeinstall_64/usr/include/lldb
+     _fakeinstall_64 fakeinstall_64/usr/lib/liblldb*
+     _fakeinstall_64 fakeinstall_64/usr/lib/python"$_pythonver"/site-packages/lldb 
+     _fakeinstall_64 fakeinstall_64/usr/share/man/man1/lldb*
+    install -Dm644 "$srcdir"/llvm-project/lldb/LICENSE.TXT "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+
+}
+
+package_lld-git(){
+   pkgdesc="Linker from the LLVM project"
+   url="https://lld.llvm.org"
+   depends=("gcc-libs")
+   provides=("lld=$pkgver")
+   conflicts=('lld')
+
+   _fakeinstall_64 fakeinstall_64/usr/include/lld
+   _fakeinstall_64 fakeinstall_64/usr/lib/liblld*
+   _fakeinstall_64 fakeinstall_64/usr/lib/cmake/lld/
+   _fakeinstall_64 fakeinstall_64/usr/bin/lld*
+   _fakeinstall_64 fakeinstall_64/usr/bin/*lld
+   _fakeinstall_64 fakeinstall_64/usr/bin/wasm-ld
+   install -Dm644 "$srcdir"/llvm-project/lld/docs/ld.lld.1 "$pkgdir/usr/share/man/man1"
+   install -Dm644 "$srcdir"/llvm-project/lld/LICENSE.TXT "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+}
+
+package_polly-git() {
+    pkgdesc="Polly is a high-level loop and data-locality optimizer and optimization infrastructure for LLVM"
+    url="https://polly.llvm.org/"
+    arch=('any')
+    depends=("llvm-git=$pkgver-$pkgrel")
+    _fakeinstall_64 fakeinstall_64/usr/include/polly
+    _fakeinstall_64 fakeinstall_64/usr/lib/cmake/polly/
+    _fakeinstall_64 fakeinstall_64/usr/lib/*Polly*
+    _fakeinstall_64 fakeinstall_64/usr/lib/libGPU*
+    install -Dm644 "$srcdir"/llvm-project/polly/LICENSE.txt "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+    _fakeinstall_64 fakeinstall_64/usr/share/man/man1/polly*
+}
+package_compiler-rt-git() {
+  pkgdesc="Compiler runtime libraries for clang (git version)"
+  url="https://compiler-rt.llvm.org/"
+  depends=("llvm-git=$pkgver-$pkgrel" 'gcc-libs')
+  provides=("compiler-rt")
+  conflicts=('clang-svn')
+
+   local _llvmver=$(echo "$pkgver" | grep -Po "(^[\.\d]+)")
+
+  _fakeinstall_64 fakeinstall_64/usr/lib/clang/"$_llvmver"/lib/linux
+  _fakeinstall_64 fakeinstall_64/usr/lib/clang/"$_llvmver"/include/{sanitizer,xray}
+  _fakeinstall_64 fakeinstall_64/usr/lib/clang/"$_llvmver"/share
+
+  install -Dm644 "$srcdir"/llvm-project/clang/LICENSE.TXT "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+}
+
+package_clang-git() {
+     pkgdesc="C language family frontend for LLVM (git version)"
+     url="http://clang.llvm.org/"
+     depends=("llvm-git=$pkgver-$pkgrel" 'gcc' "python" 'python2')
+     optdepends=('llvm-libs: for compiling with -flto')
+     provides=("clang" 'clang-analyzer' 'clang-tools-extra')
+     conflicts=('clang' 'clang-svn' 'clang-analyzer' 'clang-tools-extra' 'clang-tools-extra-svn')
+     replaces=('clang-svn' 'clang-analyzer-svn' 'clang-tools-extra-svn' 'clang')
```

Comment:
**Do not** replace `clang` or any other non-vcs packages.

Same below for: `llvm-libs`, `llvm-ocaml`, `lib32-llvm-libs`, `lib32-clang`, `lib32-llvm`.

Reply:
haha, I guess the CI caught some of these at least =)